lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -99,6 +99,18 @@
     required: false
     default: null
     version_added: "2.3"
+  delay:
+    description:
+    - Number of seconds to wait
+    required: false
+    default: 5
+    version_added: "2.3"
+  repeat:
+    description:
+    - The number of times to retry get_stack_facts operation
```

Comment:
Can you add more context to these docs about why a user would want to do this? What problems might they encounter that would need this value adjusted?